genai-perf/genai_perf/wrapper.py

```diff
@@ -113,17 +113,18 @@ def build_cmd(args: Namespace, extra_args: Optional[List[str]] = None) -> List[s
 
         utils.remove_file(args.profile_export_file)
 
+        # TODO: Temp, remove below change
```

Comment:
just a placeholder comment (to not forget to remove)

Reply:
Thanks! Removed.